packages/store/src/selection/selection-global-strategy.ts

```diff
@@ -0,0 +1,36 @@
+import { Observable, of, throwError } from 'rxjs';
+import { Injectable } from '@angular/core';
+import { tap } from 'rxjs/operators';
+
+import { NgxsConfig } from '../symbols';
+import { SelectorStrategy } from './symbols';
+
+@Injectable()
+export class SelectionGlobalStrategy implements SelectorStrategy {
+  constructor(private _config: NgxsConfig) {}
+
+  /**
+   * The default strategy defines the standard behavior of the NGXS:
+   * unsubscribe at the first error in the stream.
+   * @param errors
+   */
+  public retryWhenHandler(errors: Observable<Error | any>): Observable<Error | any> {
+    return errors.pipe(
+      tap(error => {
+        throw error;
```

Comment:
P.S. - I'm not a fan of using `tap` in that place. To avoid any such effects it's much better to map to the error stream:
```ts
return errors.pipe(
  mergeMap(error => throwError(error))
);
```
Using `throwError` - we primarily insure our users.
```

Reply:
Good